test/metabase/cmd/env_var_dox_test.clj

```diff
@@ -2,12 +2,12 @@
   (:require [clojure.test :refer :all]
             [metabase.cmd.env-var-dox :as sut]))
 
-(def settings '({:description "Have we sent an abandonment email to the instance admin?",
+(def settings '({:description "Have we sent a follow up email to the instance admin?",
```

Comment:
Note the behaviour of this test had nothing to do with the actual abadonment email setting as the function under test, `sut/format-env-var-docs`, is pure. I just thought it would be better to nice to replace it since these are all real settings.